client/scss/components/_promo.scss

```diff
@@ -142,6 +142,10 @@
   left: 0;
   padding: map-get($gutter-width, 'small')/2;
 
+  .promo--on-cream & {
```

Comment:
While I like the idea of matching the surround, it does seem overly complex. For now let's go wtih the white background but hold this discussion for Tomi.